app/src/main/res/drawable/ic_left_arrow.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="24dp" android:viewportHeight="129.0"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_left_arrow` appears to be unused
 Unused resources make applications larger and slow down builds.

Reply:
em dùng cho pull sau